website/src/app/kb/authenticate/entra/readme.mdx

```diff
@@ -0,0 +1,424 @@
+  <Image
+    src="/images/kb/authenticate/entra/1-home.png"
+    alt="App registrations"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Next, click **New registration**.
+
+<Link href="/images/kb/authenticate/entra/2-new-app.png" target="_blank">
+  <Image
+    src="/images/kb/authenticate/entra/2-new-app.png"
+    alt="New registration"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Enter **Firezone Connector** for the application name.
+
+Select **Single tenant** for the supported account types.
+
+Copy the first Redirect URI shown in the setup page from the Firezone admin
+portal and paste it into the **Redirect URI** field, ensuring **Web** is
+selected.
+
+Then, click **Register**.
+
+<Link
+  href="/images/kb/authenticate/entra/3-app-registration-information.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/entra/3-app-registration-information.png"
+    alt="App registration info"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+On the next page, click **Authentication** in the left sidebar.
+
+<Link href="/images/kb/authenticate/entra/4-app-info.png" target="_blank">
+  <Image
+    src="/images/kb/authenticate/entra/4-app-info.png"
+    alt="App info"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+In the **Web** section, click **Add URI**.
+
+<Link
+  href="/images/kb/authenticate/entra/5-authentication-add-uri.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/entra/5-authentication-add-uri.png"
+    alt="Auth add URI"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Enter the second Redirect URI shown in the setup page from the Firezone admin
+portal and click **Save**.
+
+**Important**: Ensure both Redirect URI fields are populated correctly.
+
+<Link
+  href="/images/kb/authenticate/entra/6-redirect-uri-and-save.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/entra/6-redirect-uri-and-save.png"
+    alt="redirect URI and save"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+### Step 3: Create client credentials
+
+Now you'll need to add OAuth credentials to allow Firezone to connect to your
+Entra ID account.
+
+Back on the **Firezone Connector** app registration page, click **Certificates &
+secrets** in the left sidebar.
+
+<Link
+  href="/images/kb/authenticate/entra/7-certificates-and-secrets.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/entra/7-certificates-and-secrets.png"
+    alt="certs and secrets"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Click **New client secret**.
+
+<Link
+  href="/images/kb/authenticate/entra/8-new-client-secret-button.png"
+  target="_blank"
+>
+  <Image
+    src="/images/kb/authenticate/entra/8-new-client-secret-button.png"
+    alt="new client secret button"
+    width={1200}
+    height={1200}
+  />
+</Link>
+
+Enter **Firezone Connector** for the description, then select an appropriate
+lifetime for the secret and click **Add**.
```

Comment:
Wondering if we should add a small info section here saying something like "Please be aware that because MS doesn't offer indefinite lifetime secrets, you will be responsible for rotating this secret as needed.  If the token expires, directory sync will stop and all users authenticating with MS Entra will not be able to login...."